chapters/functions.tex

```diff
@@ -2372,6 +2373,8 @@ \subsection{Annotations for External Functions}\label{annotations-for-external-l
   In case calls to several external functions are generated in the same translation unit, the \lstinline!Include! annotations of the different functions must not define the same function -- except when relying on the deprecated behavior.
 
   The included code should be valid C89 code.
+  If the code uses arrays the tool is responsible for ensuring that a C-header defining \lstinline[language=C]!size_t! is included before this code.
+  The user is responsible for ensuring that the C-code can be compiled separately -- with \lstinline[language=C]!#include <stddef.h>! prepended if using any array.
```

Comment:
We always use the spaced form of _Modelica code_, and should do the same for C:
```suggestion
  The user is responsible for ensuring that the C code can be compiled separately -- with \lstinline[language=C]!#include <stddef.h>! prepended if using any array.
```

Reply:
Also, there should be no need to spell out that it is the user who is responsible; only the user can be responsible for the annotation:
```suggestion
  The \lstinline!"insertedCode"! (with \lstinline[language=C]!#include <stddef.h>! prepended when this would be a tool responsibility) shall be possible to compile separately.
```

Reply:
I'm afraid that this formulation gives the impression that it is _stddef.h_ that tools are expected to use, so maybe we should be more careful if we want to avoid code becoming dependent upon this particular header…

Reply:
Like this:
```suggestion
  The \lstinline!"insertedCode"! (following the inclusion of a header for \lstinline[language=C]!size_t! when this would be a tool responsibility) shall be possible to compile separately.
```